roles/custom/matrix-client-element/templates/style.json.j2

```diff
@@ -0,0 +1,18 @@
+{
+    "layers": [
+        {
+            "id": "locallayer",
+            "source": "localsource",
+            "type": "raster"
+        }
+    ],
+    "sources": {
+        "localsource": {
+            "attribution": {{ matrix_client_element_location_map_attribution|to_json }},
+            "tileSize": 256,
+            "tiles": {{ matrix_client_element_location_map_tile_servers|to_json }},
```

Comment:
It's more consistent if we name this `matrix_client_element_location_sharing_styles_sources_localsource_tiles`.

Reply:
I have renamed it to `matrix_client_element_location_sharing_map_style_content_sources_localsource_tiles`.